hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java

```diff
@@ -52,6 +54,19 @@
 public abstract class MapOperation extends AbstractNamedOperation
         implements IdentifiedDataSerializable, ServiceNamespaceAware {
 
+    private static final int DEFAULT_FORCED_EVICTION_RETRY_COUNT = 5;
+    private static final String PROP_FORCED_EVICTION_RETRY_COUNT
+        = "hazelcast.internal.forced.eviction.retry.count";
+    private static final HazelcastProperty FORCED_EVICTION_RETRY_COUNT
+        = new HazelcastProperty(PROP_FORCED_EVICTION_RETRY_COUNT,
+                                DEFAULT_FORCED_EVICTION_RETRY_COUNT);
+    private static final Eviction[] EVICTIONS = new Eviction[]{
+        new RecordStoreForcedEviction(),
+        new PartitionRecordStoreForcedEviction(),
+        new AllEntriesEviction(),
+        new PartitionAllEntriesEviction()
+    };
```

Comment:
can we hide all this forced eviction specific fields inside a class, maybe in `WithForcedEviction` class which is deleted in this PR?